bundle/manifests/fence-agents-remediation.clusterserviceversion.yaml

```diff
@@ -159,7 +159,7 @@ spec:
     - Speed - FAR is rapid since it can reboot a node and receive an acknowledgment from the API call while other remediators might need to wait a safe time till they can expect the node to be rebooted.
     - Diversity - FAR includes several fence agents from a large known set of upstream fencing agents for bare metal servers, virtual machines, cloud platforms, etc.
     - Adjustability - FAR allows to set up different parameters for running the API call that remediates the node.
-  displayName: Fence Agents Remediation Operator
+  displayName: Fence Agents Remediation Operator - Community Edition
```

Comment:
Is this intended? I would imagine that we want to append `- Community Edition` only when building the bundle for releasing it to the community.

Reply:
I think I run too fast here. The idea is to have the k8s bundle committed upstream, _however_, this will appear also downstream since FAR does not have (yet) the OCP bundle built from scratch downstream.
So, for now I need to revert this change, but in the future it will be right :)

Reply:
>The idea is to have the k8s bundle committed upstream, however, this will appear also downstream since FAR does not have (yet) the OCP bundle built from scratch downstream.

I wasn't aware of that idea. I understand why it is needed for NHC, but not sure we have discussed it widely for the rest of the operators. It might be an improvement, not sure.

Reply:
Why should be good for one operator and not for the others?

Reply:
IIUC the idea for using that for NHC was due to three different bundle creation options: For k8s, for metrics, and for OCP. This scenario is unique since only NHC has a metrics bundle variant. Unless I am wrong here and it is planned to be created for each Medik8s operator.

Reply:
My understanding was to have the "pure" k8s bundle upstream regardless the other options, but I might be wrong and in any case downstream CI isn't ready for FAR to support it anyway :)